common/gtk-3.0/3.18/render-asset.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -ueo pipefail
+
+INKSCAPE="$(which inkscape)"
+OPTIPNG="$(which optipng)"
+
+SRC_FILE="assets.svg"
+ASSETS_DIR="assets"
+
+i="$1"
+
+echo "Rendering '$ASSETS_DIR/$i.png'"
+"$INKSCAPE" --export-id="$i" \
+            --export-id-only \
+            --export-png="$ASSETS_DIR/$i.png" "$SRC_FILE" >/dev/null \
+&& "$OPTIPNG" -o7 --quiet "$ASSETS_DIR/$i.png"
+
+echo "Rendering '$ASSETS_DIR/$i@2.png'"
```

Comment:
Why are you rending the image twice?

Reply:
it's used automatically on hi-density displays (on non-gtk3 they can't be used at the same time and need to be generated instead, but pseudo-hidpi on gtk2 not implemented yet)

Reply:
(you observe the resulting generated css files after the full build)